modules/pipeline/providers/clusterinfo/interface.go

```diff
@@ -91,6 +95,12 @@ func (p *provider) BatchUpdateAndDispatchRefresh() error {
 }
 
 func (p *provider) batchUpdateClusterInfo() error {
+	currentClusterInfo, err := p.GetCurrentClusterInfo()
```

Comment:
Is it wrong here? for batch update.

Reply:
edge and central deploy both need to batch update cluster info
edge deploy pipeline need to update self cluster at least